unlock-app/src/components/content/CheckoutContent.tsx

```diff
@@ -26,16 +24,6 @@ interface CheckoutContentProps {
 
 const defaultLockAddresses: string[] = []
 
-const lockKeysAvailable = (lock: RawLock) => {
```

Comment:
I changed this so we pass the whole lock object to the `Lock` component (so we don't bloat the props with the things needed for the hook), so all of this is now handled internally.